lib/xeroizer/oauth.rb

```diff
@@ -37,20 +37,20 @@ class UnknownError < StandardError; end
         :request_token_path => "/oauth/RequestToken",
         :access_token_path  => "/oauth/AccessToken",
         :authorize_path     => "/oauth/Authorize",
-        :ca_file            => File.expand_path(File.join(File.dirname(__FILE__), 'ca-certificates.crt'))
+        #:ca_file            => File.expand_path(File.join(File.dirname(__FILE__), 'ca-certificates.crt'))
```

Comment:
They are no longer using certificates for authentication, but I am not sure why they just commented it out instead of deleting it.